packages/shared/lib/services/providers.ts

```diff
@@ -39,7 +40,7 @@ function getProvidersPath() {
     return findProvidersYaml(dirname());
 }
 
-function loadProvidersYaml(): Record<string, Provider> | undefined {
+export function loadProvidersYaml(): Record<string, Provider> | undefined {
```

Comment:
does this need to be exported?

Reply:
Oh nope, that was from a previous idea of how to do this. Removing!

Reply:
Fixed in cb23febc